3737/2/model/user_class.rb

```diff
@@ -0,0 +1,15 @@
+require 'ohm'
+
+# user model
+class User < Ohm::Model
+  attribute :number
+  attribute :in_camp
+
+  def self.registered(id)
```

Comment:
you have more than 1  class method so better to use construction `class << self`

Reply:
Should this method return boolean value (whether student is registered)? If so then add '?'